scripts/build_schemas.sh

```diff
@@ -8,13 +8,19 @@ for region_code in GB-WLS GB-ENG GB-NIR; do
     FORMATTED_REGION_CODE=$(echo "${region_code}" | tr '[:upper:]' '[:lower:]' | tr - _)
     DESTINATION_FILE="data-source/jsonnet/census_individual_${FORMATTED_REGION_CODE}.json"
 
+    if [[ "$region_code" = "GB-WLS" ]]; then
+        CENSUS_DATE="2019-06-20"
```

Comment:
As mentioned on slack, these dates don't need to be different

Reply:
The card only referenced Wales, but I'm told that's not the case, so I've removed the conditional code and updated the date to the 20th.
What I didn't know is that the Welsh schema wasn't also built automatically. Alas!

Reply:
Built the schema.